internal/future/storage/public.go

```diff
@@ -30,6 +31,11 @@ var (
 // Service is a generic interface for talking to a storage backend.
 type Service interface {
 	// Close shutsdown the database connection.
+	CreateTicket(ticket pb.Ticket, ttl int) error
```

Comment:
Why are we passing in a TTL? Can we just have the DB client know what a reasonable default TTL is from loading the configuration at load? Or saving a cfg instance and reading it in place (that way config pushes take)

Reply:
There are a few values (ttl, indexes etc.) that I am currently designing for state store to take externally than depending on the cfg. State storage taking dependency on config for its runtime seems strange (do you think it should store the config view and reach out to the config itself?)